snap/snapcraft.yaml

```
@@ -66,8 +66,8 @@ description: |
   **Cryptography**
   ---
 
-  The snap installs the CUDA keyring, verifying its integrity using `curl` and SHA256 checksums.
-  The keyring also sets up the appropriate source for the DCGM deb package.\
+  The snap installs the CUDA keyring deb package using `curl` and verifies its integrity using SHA256 checksums.
+  During the snap build process, the keyring sets up the appropriate source for the DCGM deb package and verifies its signature.\
```

```suggestion
  During the snap build process, the install scripts of the CUDA keyring deb package set up the appropriate source for the DCGM deb package, whose signature is then verified by using the keyring itself.\
```
...although I would suggest rephrasing the whole paragraph so it's clearer that all of this crypto only happens during building